sdk/storage/azure-storage-blobs/CHANGELOG.md

```diff
@@ -6,6 +6,10 @@
 
 - Added support for customized application ID.
 
+### Breaking Changes
+
+- Changed the return type of `StartCopyFromUri` and `StartCopyIncremental` API to `StartCopyBlobOperation` as suggested by core sdk team. There's no way to access raw response anymore, although this is a clear violation to the entity of [exposing raw response](https://azure.github.io/azure-sdk/cpp_introduction.html#cpp-design-logical-client-expose-raw) in C++ Guidelines.
+
```

Comment:
hahahaha,, I don't think customers want to know about this =D

```suggestion
- Changed the return type of `StartCopyFromUri` and `StartCopyIncremental` API to `StartCopyBlobOperation`.

```

Reply:
Operation<T> will have a `GetRawResponse()` before we GA,
and calling `Poll()` will return the status instead of the `rawResponse`